app/screens/settings/display_settings/display_settings.test.js

```diff
@@ -0,0 +1,33 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+import React from 'react';
+import {shallow} from 'enzyme';
+import SettingsItem from 'app/screens/settings/settings_item';
+import Preferences from 'mattermost-redux/constants/preferences';
+
+import DisplaySettings from './display_settings';
+
+jest.mock('react-intl');
+
+describe('DisplaySettings', () => {
+    const baseProps = {
+        theme: Preferences.THEMES.default,
+        enableTheme: false,
+        enableTimezone: false,
+        navigator: {push: () => {}}, // eslint-disable-line no-empty-function
```

Comment:
Doesn't need to be changed now, but we can use `jest.fn()` in the future. Alternatively, we could just disable this rule in tests since it's not useful here